src/indexed-formula.js

```diff
@@ -62,7 +62,7 @@ function handleRDFType (formula, subj, pred, obj, why) {
 }
 
 class IndexedFormula extends Formula {
-  constructor (features) {
```

Comment:
Why remove the spaces? (standard formatting requires the space in the function declaration)